tests/requirements-linting.txt

```diff
@@ -1,6 +1,7 @@
 black==20.8b1
 flake8==3.8.4
 flake8-quotes==3.2.0
+hypothesis==5.44.0
```

Comment:
Should.be removed from linting deps imo

Reply:
Other than this looks great to me!

Reply:
Done!

Reply:
Alas, `mypy` gives different results if it's installed - so I thought it was better to always have it than to leave it out in CI but have errors if people ran `mypy pydantic` locally with Hypothesis installed.

Reply:
Ah! Ok it makes sense I agree with you